ccdImports/configFiles/CCD_Probate_Legacy_Search/AuthorisationCaseEvent.json

```diff
@@ -2,5 +2,8 @@
   {"CaseTypeID": "LegacySearch", "CaseEventID": "createDummy", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
   {"CaseTypeID": "LegacySearch", "CaseEventID": "legacySearch", "UserRole": "caseworker-probate-issuer", "CRUD": "CRUD"},
   {"CaseTypeID": "LegacySearch", "CaseEventID": "createDummy", "UserRole": "caseworker-probate-caseadmin", "CRUD": "CRUD"},
-  {"CaseTypeID": "LegacySearch", "CaseEventID": "legacySearch", "UserRole": "caseworker-probate-caseadmin", "CRUD": "CRUD"}
+  {"CaseTypeID": "LegacySearch", "CaseEventID": "legacySearch", "UserRole": "caseworker-probate-caseadmin", "CRUD": "CRUD"},
```

Comment:
I suggest we take this opportunity to remove CU access for ordinary users for create dummy event
And make it only CRUD for superusers

Reply:
talked to stephen about this